RELEASE-NOTES.md

```diff
@@ -14,6 +15,28 @@ Contents
  * [v0.10](#v0.10)
  * [v0.8](#v0.8)
 
+v0.19 <a name="v0.19"></a>
+-----
+
+#### Core
+
+ * Report 
+ * Fix bug where acquisition could get stuck searching an empty doppler range
+ * Fix buffer overflows when building NMEA messages
+ * Add better checks and thread safety to random number generator
+ * Add GPRMC, GPVTG, and GPGLL NMEA messages
+ * Settings and FileIO messages are now directional
+
+#### libswiftnav
+
+ * RAIM position outlier filtering
```

Comment:
Maybe "Receiver Autonomous Integrity Monitoring (RAIM) to filter out bad position solutions"?